Makefile

```diff
@@ -185,7 +120,7 @@ distclean: clean
 .PHONY: docker-build
 docker-build:
 	@echo "Building docker image: $(DOCKER_TAG)"
-	@-docker build "$(APP_ROOT)" -t "$(DOCKER_TAG)"
+	@-docker build . -t "$(DOCKER_TAG)"
```

Comment:
Using `.` instead of `APP_ROOT` resolved to 'current makefile directory' with `APP_ROOT := $(abspath $(lastword $(MAKEFILE_LIST))/..)` causes this (and other targets using `.`) to fail when calling `make -f path/to/Makefile <target>` from another directory.

Reply:
but it is working with `make -C /path/to/twitcher`.